app/assets/json/translations/react/es.json

```diff
@@ -1213,7 +1220,9 @@
   "lottery.enterLotteryNo": "Ingrese su número de sorteo",
   "lottery.enterLotteryNumber": "Ingrese su número de sorteo",
   "lottery.generalPool": "Pozo general",
+  "lottery.generalPublic": "Público general",
```

Comment:
What code should I use to see this string? Looking at the doc I'm not sure

Reply:
Same thing here as the other comment about `generalPublic`